app/views/user_mailer/_request_expired.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>Your <%= link_to 'reservation request', reservation_url(@reservation, only_path: false) %> has expired.<p>
```

Comment:
Whoops, we have some typos here (I think) - this close tag is missing a backslash and the next line is both missing a period after the link and a closing `</p>` tag.